spec/features/initiative_spec.rb

```diff
@@ -40,6 +40,12 @@
     expect(page).to have_content('TestB')
     expect(page).to_not have_content('TestA')
   end
+  scenario 'delete initiative from list view' do
+    visit root_path
+    expect{@initiative.destroy}.to change(Initiative,:count).by(-1)
+    expect(page).to have_content('Initiatives')
```

Comment:
What kind of test could we do here to make sure the deleted initiative is _not_ on the page?

Reply:
We updated the feature test to ensure initiative deletion is not shown on the page